### PR TITLE
Page

### PR DESCRIPTION
**Pull request by andy-faunadb**

### CODE REVIEW

File: lib/fauna/page.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    #     end
+    #   end)
+    #   puts iter.to_a
+    #
+    # +client+:: A Client.
+    # +set_query+:: Set query to paginate, such as Query.match.
+    # +params+:: All optional.
+    #            +:map+:: Query.lambda for mapping set elements.
+    #            +:page_size+:: Number of instances to be fetched per page.
+    def self.set_iterator(client, set_query, params = {})
```
Comment (andy-faunadb) [inline lib/fauna/page.rb]: Maybe this should be called `instance_iterator`.

Comment (freels) [inline lib/fauna/page.rb]: It's iterating over tuples including covered values. instance is a bit misleading.

Comment (andy-faunadb) [inline lib/fauna/page.rb]: I'm looking for something complementary to `page_iterator`. Maybe `element_iterator`?

**Comment by ashfire908** [comment]
Replaced by #150.

**Pull request closed without merging**